AmplifyTestCommon/Mocks/MockAPICategoryPlugin.swift

```diff
@@ -178,6 +178,16 @@ class MockAPICategoryPlugin: MessageReporter, APICategoryPlugin {
             return operation
     }
 
+    @available(iOS 13.0, *)
+    public func reachabilityPublisher(for apiName: String? = nil) -> AnyPublisher<ReachabilityUpdate, Never>? {
```

Comment:
As above, remove default arg now that we have a no-arg method

Reply:
thanks.